### PR TITLE
Fixed badges and codacy issues

### LINKED ISSUE #249: Fix readme

### PR DESCRIPTION
**Pull request by elheck**
closes #249 

This is also a test if the automated toolchain still works

### CODE REVIEW

**Comment by elheck** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/SquidSquad/Squiddrone/pullRequest?prid=5810451&bid=19020139)

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-[![badge](https://report.ci/status/elheck/Squiddrone/badge.svg?branch=dev)](https://report.ci/status/elheck/Squiddrone?branch=dev)
-[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/elheck/Squiddrone/tree/dev) 
+[![Build Status](https://travis-ci.com/Squiddrone/Squiddrone.svg?branch=dev)](https://travis-ci.com/Squiddrone/Squiddrone)
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/88da9a5c932545adb22b885a39a1ae35)](https://www.codacy.com/manual/SquidSquad/Squiddrone?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=elheck/Squiddrone&amp;utm_campaign=Badge_Grade)
+[![CodeFactor](https://www.codefactor.io/repository/github/squiddrone/squiddrone/badge)](https://www.codefactor.io/repository/github/squiddrone/squiddrone)
+[![codecov](https://codecov.io/gh/elheck/Squiddrone/branch/dev/graph/badge.svg)](https://codecov.io/gh/Squiddrone/Squiddrone)
+[![badge](https://report.ci/status/elheck/Squiddrone/badge.svg?branch=dev)](https://report.ci/status/Squiddrone/Squiddrone?branch=dev)
```
Comment (ThorsHamster) [inline README.md]: There are still some mentions of "elheck" here. Please replace it with "Squiddrone".

**Review by ThorsHamster — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (elheck) [reply README.md]: Done

**Comment by elheck** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/SquidSquad/Squiddrone/pullRequest?prid=5810451&bid=19020139)

**Review by ThorsHamster — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**